pkgs/tools/inputmethods/fcitx5/fcitx5-rime.nix

```diff
@@ -6,6 +6,9 @@
 , gettext
 , fcitx5
 , librime
+, rime-data
+, symlinkJoin
+, rimeData ? [ rime-data ]
```

Comment:
Suggest a better name like `rimeDataPkgs` to avoid confusion.

Reply:
Thank you for your suggestion! In the NixOS CN Matrix groups, there are many rime users, we also discussed this problem, many names are suggested:

- `rimeData` (causing confusion with `rime-data`)
- `rimeDataPackages` (too long)
- `rimeDataPkgs` (more acceptable than `rimeDataPackages`)
- `rimeDicts` (rime data include not only dictionaries but schemas and OpenCC configs, etc)
- `rimeSchemas` (same problem as `rimeDicts`)
- `dataPackages`, `dataPkgs` (confusion, what data?)

I think `rimeDataPkgs` should be a good choice, I have changed the name of the argument to `rimeDataPkgs`.